com.playeveryware.eos/Runtime/Core/Config/SandboxId.cs

```diff
@@ -38,6 +38,13 @@ readonly get
             }
             set
             {
+                // allow null value
+                if (value == null)
```

Comment:
Given that the string is the value provided, could this use "IsNullOrEmpty", and sanitize that to a standard either empty string or null?

Reply:
Ah, that's worthwhile. I sort of feel like the correct approach might to be treating null and empty as entirely interchangeable, but I don't feel strongly about it. I'm leaving his comment unresolved so you can see it, but resolve it after you do.